app/notes/getNotes.tsx

```diff
@@ -0,0 +1,42 @@
+import pb from "@/app/lib/pocketbase";
+import Link from "next/link";
+import * as querystring from "node:querystring";
+
+export const dynamic = 'auto',
+    dynamicParams = true,
+    revalidate = 0,
+    fetchCache = 'auto',
+    runtime = 'nodejs',
+    preferredRegion = 'auto'
+
+export async function getNotes(storedUser:string|null) {
+    if (!storedUser) {
```

Comment:
if you use Pocketbase for learning purposes I think crashalytics from firebase is free to a certain target, ty and maybe make a logger class that you call and it does not only log to firebase but also to console.error